Source/Core/DolphinQt/Debugger/NetworkWidget.cpp

```diff
@@ -90,6 +90,15 @@ QTableWidgetItem* GetSocketState(s32 host_fd)
   return new QTableWidgetItem(QTableWidget::tr("Unbound"));
 }
 
+QTableWidgetItem* GetSocketBlocking(s32 wii_fd)
+{
+  const auto& socket_manager = IOS::HLE::WiiSockMan::GetInstance();
+  if (socket_manager.GetHostSocket(wii_fd) < 0)
+    return new QTableWidgetItem();
```

Comment:
Not saying it should be removed, but why are you checking for a host socket?

Reply:
The function is called for each possible socket fd. Some are used by the Wii and some aren't. The one which aren't will return `-EBADF`, that's why I'm checking the GetHostSocket return value. The same check is done in many get functions above: `if (host_fd < 0)`.